persistence/4.0/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Jakarta Persistence 4.0"
+date: 2025-04-14
+summary: "Release for Jakarta EE 12"
+---
+Jakarta Persistence defines a standard for management of persistence
+and object/relational mapping in Java(R) environments.
+
+### New features, enhancements or additions
+
+- Introduce a new API exposing persistence operations for use with detached entities not associated with a persistence context, along with associated lifecycle callbacks
+- Introduce a programmatic API for expressing native query result set mappings
+- More elegant annotations for expressing named entity graphs
+- Allow entities to be loaded in read-only mode
+- Add `findMultiple()` to `EntityManager`
+- Add `getResultCount()` to `Query`
+- Add an annotation to exclude a field from optimistic locking
+- Allow `@NamedQuery` and `@NamedNativeQuery` more control over query execution
+- Allow passing an `EntityGraph` to `refresh()`
+- Make it easier to obtain and apply entity graphs
+- Make `select new` implicit when a query result type is given
+- Add a `@PreMerge` lifecycle callback
+- Add support for `SequencedCollection` and friends
+- Decouple entity discovery/enhancement from `EntityManagerFactory` initialization in container environments
```

Comment:
This proposal is described here: https://github.com/jakartaee/persistence/issues/650